packages/core/src/awsService/cloudWatchLogs/commands/tailLogGroup.ts

```diff
@@ -99,6 +103,24 @@ function formatLogEvent(logEvent: LiveTailSessionLogEvent): string {
     return line
 }
 
+//Auto scroll visible LiveTail session editors if the end-of-file is in view.
+//This allows for newly added log events to stay in view.
+function getTextEditorsToScroll(document: vscode.TextDocument): vscode.TextEditor[] {
+    return vscode.window.visibleTextEditors.filter((editor) => {
+        const isEditorForSession = editor.document === document
+        if (!isEditorForSession) {
+            return false
+        }
+        return editor.visibleRanges[0].contains(new vscode.Position(document.lineCount - 1, 0))
+    })
+}
+
+function scrollTextEditorToBottom(editor: vscode.TextEditor) {
+    const topPosition = new vscode.Position(Math.max(editor.document.lineCount - 2, 0), 0)
+    const bottomPosition = new vscode.Position(Math.max(editor.document.lineCount - 2, 0), 0)
```

Comment:
These two constants are the same, are we able to just have one that's scroll position and then pass that two times into range?

Reply:
Yeah definitely!